Frontend/Assignments/GitNotes/README.md

```diff
@@ -12,6 +12,10 @@ Everyone takes notes and sometimes people want to share it with other people, es
   - `Login` menu for login with github
   - `Search` for searching a Note using ID
 
+![Grid View](https://user-images.githubusercontent.com/15946354/62457522-7b1c3600-b794-11e9-9fc1-82e1c5b68828.png)
```

Comment:
Please at least explain the screen what this screen about and guide the men-tee that what we expect in this screen

Reply:
I've explained them in pages flow & mocks are designed relative to that.

Reply:
I've updated the content & order of the mocks for better explanation

Reply:
Great